content/_index.md

```diff
@@ -25,23 +25,41 @@ OSM runs an Envoy based control plane on Kubernetes, can be configured with SMI
 
 ## Install
 
+![OSM Install Demo](https://github.com/openservicemesh/osm/raw/release-v0.8/img/osm-install-demo-v0.2.0.gif "OSM Install Demo")
```

Comment:
I don't know if it's a good plan to embed this for a few reasons:
- this is being embedded from the code repo so it could disappear without warning
- despite being in the release branch for v0-8 and being inline in the README on main, this is a demo of v0.2
- Is there going to be a v0.9 version of this? If not, we're creating confusion by adding already-outdated content to the main branch.

Reply:
removed